app/validation.py

```diff
@@ -176,3 +178,17 @@ def is_valid_service_id_or_400(service_id):
         return True
     else:
         abort(400, "Invalid service ID supplied: %s" % service_id)
+
+
+def is_valid_date(date, default_format='%Y-%m-%d'):
```

Comment:
Would it be worth setting this as DATE_FORMAT in [dmutils.formats](https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/formats.py)?

<sub>I've [used the same format](https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/status.py#L31) for our FeatureFlags, so it might be a good idea.</sub>